message_sender/models.py

```diff
@@ -12,6 +12,7 @@ class Channel(models.Model):
     VUMI_TYPE = 'vumi'
     JUNEBUG_TYPE = 'junebug'
     HTTP_API_TYPE = 'http_api'
+    WASSUP_API_TYPE = 'wassup'
```

Comment:
Should we not also add this to `CHANNEL_TYPES`, and make a migration for it?

Reply:
good point, will add.